src/tss.c

```diff
@@ -786,9 +786,7 @@ int tss_request_add_se_tags(plist_t request, plist_t parameters, plist_t overrid
 		return -1;
 	}
 
-	/* add tags indicating we want to get the SE,Ticket */
 	plist_dict_set_item(request, "@BBTicket", plist_new_bool(1));
-	plist_dict_set_item(request, "@SE,Ticket", plist_new_bool(1));
```

Comment:
Isn't this entry required to get the SE Ticket ?

Reply:
Oh I see, you add the DeviceGeneratedRequest so it would set this value for us. Regardless, I wonder if we break older versions with this, we'd need to know if DeviceGeneratedRequest has always been there...

Reply:
Maybe we can add the SE ticket in case we don't have any DeviceGeneratedRequest instead of returning an error (something like throwing a warning and setting the p_dgr to the SE,Ticket as a fallback)